test/db/formats/mach0/objc

```diff
@@ -287,18 +287,18 @@ add fp, sp, 0x50
 adrp x8, section.13.__DATA.__objc_const
 add x8, x8, 0x150
 str x8, [var_50h]
-stur w0, [var_14h]
-stur x1, [var_20h]
+stur w0, [fp, -4]
```

Comment:
I'm still trying to debug, and it seems to be because there's something wrong with the esil execution, although I don't quite understand what the changes I made have to do with esil.

Reply:
The main reason for this is that sorting the functions destroys the order in which esil executes them.

Since the first function to be executed changes the state of analysis->reg, so esil needs to be executed in some kind of order instead of sequentially by the address of the function.